internal/datastore/crdb/crdb.go

```diff
@@ -516,9 +519,25 @@ func (cds *crdbDatastore) OfflineFeatures() (*datastore.Features, error) {
 }
 
 func (cds *crdbDatastore) Features(ctx context.Context) (*datastore.Features, error) {
-	features, _, err := cds.featureGroup.Do(ctx, "", func(ictx context.Context) (*datastore.Features, error) {
+	cds.featuresLock.Lock()
+	cached := cds.cachedFeatures
+	cds.featuresLock.Unlock()
```

Comment:
technically this permission can change over time, the account spicedb is using is granted changefeed permission after starting up.

Reply:
Yes, it could, but at that point it would error anyway

Reply:
It could start without access and then gain access after startup

Reply:
Yeah, but that's also fine. We don't need to support that case; restart the pod